yarn.lock

```diff
@@ -4,7 +4,7 @@
 
 "@actions/core@^1.9.1":
   version "1.9.1"
-  resolved "https://registry.yarnpkg.com/@actions/core/-/core-1.9.1.tgz#97c0201b1f9856df4f7c3a375cdcdb0c2a2f750b"
+  resolved "https://registry.npmjs.org/@actions/core/-/core-1.9.1.tgz"
```

Comment:
There shouldn't be any `yarn.lock` changes for this PR — you'll need to revert this change.

Reply:
Thanks, that worked!